spark_parser/spark.py

```diff
@@ -160,7 +160,10 @@ def addRule(self, doc, func, _preprocess=True):
             else:
                 self.rules[lhs] = [ rule ]
             self.rule2func[rule] = fn
-            self.rule2name[rule] = func.__name__[2:]
+            if func.__name__ == '<lambda>': #STRANIKS_SCAN 29/02/2016
```

Comment:
I am lost at why the rule<lambda> should be treated differently than other rules? 

Perhaps if you give me an example, I'd understand the situation better. Thanks